packages/connect-node/src/connect-http-transport.ts

```diff
@@ -306,7 +306,7 @@ export function createConnectHttpTransport(
                     "cannot send, stream is already closed"
                   );
                 }
-                let flags = messageFlag;
+                let flags = 0;
```

Comment:
I think a straight-forward `0` is the better choice, now that we have moved all other flag constants to a proper place. It doesn't seem worth it to add a class or constructor function for EnvelopedMessage.